package.json

```diff
@@ -17,19 +18,23 @@
     "lint": "eslint --max-warnings=0 .",
     "lint-fix": "eslint --fix .",
     "test": "node test.js | tap-parser -l",
+    "build": "tsc",
```

Comment:
Still need to figure out how we want to include this output…

Reply:
@theengineear Do you need to? If you're using JSDoc, TypeScript will import the file as typed.

Reply:
Since the entire interface is really a single class — I do think it makes sense to keep this checked into source as a sibling of the file itself. It allows us to scrutinize any changes that could happen there.

That said… that’s just my current opinion there. It’s completely subject to change if it ends up being overly cumbersome!